site/profile/manifests/app/splunk/server.pp

```diff
@@ -0,0 +1,33 @@
+# splunk server module
+class profile::app::splunk::server (
+  $purge_inputs       = false,
```

Comment:
Don't think you need either purge_inputs and purge_outputs parameters. I'd say that even the src_root is unecessary.